db/seeds.rb

```diff
@@ -1,63 +1,112 @@
-20.times do
-  User.create(
-    email: Faker::Internet.email,
-    username: Faker::Name.first_name,
-    password: Faker::Internet.password(8),
+User.create(
+    email: 'pablo.ledesma.p@gmail.com',
+    username: 'Pablo',
+    password: '923138334',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.